examples/FirebaseRoom_ESP8266/README.md

```diff
@@ -25,16 +25,17 @@ sensor data to Firebase, and trigger actuators from Firebase.
 
 ## Configuration
 
-1. Go to https://www.firebase.com/ and create a new Firebase
-1. Copy the `Firebase Hostname`
-1. Go to `Secrets`
+1. Go to https://firebase.google.com/ and create a new Firebase
+1. Go to `Database`
+1. Copy the `Database hostname` (Database URL without `https://` and trailing `/`)
```

Comment:
We are asking users to Copy the hostname, but there are no instructions on where to paste it.